buggy-queries.txt

```diff
@@ -0,0 +1,2 @@
+what is the average salary of employees where name is not vinh
+how many jobs where name is not vinh
```

Comment:
Also after this PR I realized that we really need to iron out error cases - I realized our API is really error prone right now / might null pointer/ might return an unhelpful "problem querying ..." message.